Clean Code/ToDo/Program.cs

```diff
@@ -1,110 +1,156 @@
+                    return;
+                }
+
                 Console.WriteLine("Enter the number of the task to remove: ");
                 // Show current taks
-                for (int i = 0; i < TL.Count; i++)
+                for (int i = 0; i < TaskList?.Count; i++)
                 {
-                    Console.WriteLine((i + 1) + ". " + TL[i]);
+                    int j = i + 1;
+
+                    Console.WriteLine(j + ". " + TaskList[i]);
                 }
                 Console.WriteLine("----------------------------------------");
 
-                string line = Console.ReadLine();
+                string? line = Console.ReadLine();
+                if (line?.Length <= 0)
+                {
+                    Console.WriteLine("No task was removed.");
+                    return;
+                }
+
                 // Remove one position
                 int indexToRemove = Convert.ToInt32(line) - 1;
+
                 if (indexToRemove > -1)
                 {
-                    if (TL.Count > 0)
-                    {
-                        string task = TL[indexToRemove];
-                        TL.RemoveAt(indexToRemove);
-                        Console.WriteLine("Task " + task + " deleted");
-                    }
+                    string task = TaskList[indexToRemove];
+                    TaskList.RemoveAt(indexToRemove);
+
+                    Console.WriteLine("Task " + task + " deleted");
                 }
             }
-            catch (Exception)
+            catch (Exception ex)
             {
+                Console.WriteLine($"RemoveTask failed. ({ex.Message})");
             }
         }
 
-        public static void ShowMenuAdd()
+        private static void ShowMenuPendingTask()
         {
             try
             {
-                Console.WriteLine("Enter the name of the task: ");
-                string task = Console.ReadLine();
-                TL.Add(task);
-                Console.WriteLine("Task registered successfully");
-            }
-            catch (Exception)
-            {
-            }
-        }
+                if (TaskList == null || TaskList.Count == 0)
+                {
+                    Console.WriteLine("There are no tasks to perform.");
+                    return;
+                }
 
-        public static void ShowMenuTres()
-        {
-            if (TL == null || TL.Count == 0)
-            {
-                Console.WriteLine("There are no tasks to perform");
-            } 
-            else
-            {
                 Console.WriteLine("----------------------------------------");
-                for (int i = 0; i < TL.Count; i++)
+                for (int i = 0; i < TaskList.Count; i++)
                 {
-                    Console.WriteLine((i + 1) + ". " + TL[i]);
+                    int j = i + 1;
+
+                    Console.WriteLine(j + ". " + TaskList[i]);
```

Comment:
Instead using a `for` loop, you can use the `foreach` statement, it's more natural for reading. You are adding a new variable and adding more complexity to your code. Also, you should use string interpolation to have a better message. What do you think about refactoring this logic like this:
```suggestion
            var indexTask = 1;
            TaskList.ForEach(task => Console.WriteLine($"{indexTask++}. {task}"));
        }
```